books/kestrel/utilities/system/numbered-names.lisp

```diff
@@ -123,17 +126,25 @@
   "{")
 
 (define get-numbered-name-index-start ((wrld plist-worldp))
-  :returns (start "A @(tsee numbered-name-index-start-p).")
-  :verify-guards nil
+  :returns (start numbered-name-index-start-p "A @(tsee numbered-name-index-start-p).")
```

Comment:
You can remove `”A @(tsee numbered-name-index-start-p).”` now that it’s actually code.